lib/graphql/language/nodes.rb

```diff
@@ -202,7 +202,7 @@ class FragmentDefinition < AbstractNode
         def initialize_node(name: nil, type: nil, directives: [], selections: [])
           @name = name
           @type = type
-          @directives = directives
+          @directives = directives || []
```

Comment:
Good question. I did this because I saw we had done it for [`ObjectTypeDefinition` `interfaces`](https://github.com/rmosolgo/graphql-ruby/blob/1a7f560212c419f2ab2aa6020528582cc744f5a2/lib/graphql/language/nodes.rb#L346).

I was under the impression we did this so that if someone intentionally passes `nil`, it would get turned into `[]`. It does add quite a bit noise, maybe we can just remove the `|| []`.